app/features/analysis/calculate/OneRMCalculateActions.js

```diff
@@ -167,8 +167,10 @@ const logCalculate1RMAnalytics = (state, exercise, numIncludeTags, numExcludeTag
         slope: slope,
         min_weight: minWeight,
         max_weight: maxWeight,
+        weight_range: maxWeight - minWeight,
```

Comment:
- [ ] this is technically correct, however I want to make it weight_lb_range and to actually convert weights into lbs. The analytics doc will be updated accordingly in a few